javascript/modules/client/test/integration/client.test.ts

```diff
@@ -420,6 +423,57 @@ describe("Client", () => {
       expect(typeof withdrawAction).toBe("object");
       expect(withdrawAction.data).toBeInstanceOf(Uint8Array);
     });
+    it("Should create a client and generate a grant action", async () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const grantParams: IPermissionParams = {
+        who: "0x1234567890123456789012345678901234567890",
+        where: "0x1234567890123456789012345678901234567890",
+        permission: DaoPermissions.UPGRADE_PERMISSION,
+      };
+
+      const grantAction = await client.encoding.grantAction(
+        grantParams,
+      );
+
+      expect(typeof grantAction).toBe("object");
+      expect(grantAction.data).toBeInstanceOf(Uint8Array);
```

Comment:
Why don't we assert on the `to`, `value` and `data` hex stringified value?

Reply:
Could we check `bytesToHex(grantAction.data)` instead and compare it to a hardcoded hex string?